profile/2020-09-01-hybrid.json

```diff
@@ -584,67 +584,67 @@
         }
       ]
     },
-    "microsoft.containerregistry" : {
```

Comment:
Thank you for doing this.

@konrad-jamrozik since Zach didn't touch this directory, why did validation flag issues in other directories? It's been properly scoped to data-plane vs. resource-manager directories, IIRC. Is that scoping not updated to handle the new [TypeSpec project directory structure](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md)?